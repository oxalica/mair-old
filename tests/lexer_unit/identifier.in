_
a
as
asc
true false null
a0__c_
_9 a0
0a
