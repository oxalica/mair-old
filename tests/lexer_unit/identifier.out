Ok([(Ident("_"), "_")])
Ok([(Ident("a"), "a")])
Ok([(Keyword(As), "as")])
Ok([(Ident("asc"), "asc")])
Ok([(Literal(Bool(true)), "true"), (Literal(Bool(false)), "false"), (Ident("null"), "null")])
Ok([(Ident("a0__c_"), "a0__c_")])
Ok([(Ident("_9"), "_9"), (Ident("a0"), "a0")])
Err(LexicalError { loc: "0a", kind: InvalidNumberSuffix })
